src/components/BlogForm/TitleEntry.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+import {
+  Button,
+  Container,
+  Stack,
+  TextField,
+  Typography,
+  useTheme,
+} from "@mui/material";
+import { postBlog } from "api/blog";
+import { LoadingScreen } from "components/global-components";
+import _ from "lodash";
+import { useSession } from "next-auth/react";
+import { useSnackbar } from "notistack";
+import { Controller, useForm } from "react-hook-form";
+import { v4 as uuid } from "uuid";
+import { titleDataProps } from "./types";
+
+const TitleEntry = () => {
+  const theme = useTheme();
+  const { enqueueSnackbar } = useSnackbar();
+  const { data: session } = useSession();
+
+  const { control, handleSubmit } = useForm({
+    defaultValues: {
+      title: "",
+      date: "",
+    },
+  });
+
+  if (!session?.user) {
+    return <LoadingScreen />;
+  }
+
+  const { user } = session;
+
+  const onSubmit = (data: titleDataProps) => {
+    const blogSlug = _.kebabCase(data.title);
+    const blogPostId = blogSlug + uuid();
+    postBlog(blogPostId, {
+      ...data,
+      author: user.name,
+      slug: blogSlug,
+      date: new Date().toDateString(),
+    })
+      .then(() => {
+        enqueueSnackbar("Success! A new blog post has been initialised");
+        window.location.replace("http://localhost:3000/dashboard");
+      })
+      .catch((e) =>
+        enqueueSnackbar(`Error! Unable to initialise new blog post`, {
+          variant: "error",
+        })
+      );
+  };
+
+  return (
+    <Container maxWidth="lg" sx={{ height: "100%", mt: 20 }}>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Stack
+          direction={"column"}
+          spacing={4}
+          sx={{
+            alignItems: "center",
+            textAlign: "center",
+          }}
+        >
+          <Typography variant="h4">Please enter a title</Typography>
+          <Stack
+            direction={"row"}
+            spacing={4}
+            justifyContent={"space-around"}
+            minWidth={"50%"}
+          >
+            <Controller
+              name="title"
+              control={control}
+              rules={{ required: true }}
+              render={({
+                field: { onChange, value },
+                fieldState: { error },
+              }: any) => (
+                <TextField
+                  label="New Title..."
+                  fullWidth
+                  value={value}
+                  onChange={onChange}
+                  error={!!error}
+                  helperText={!!error && "Title is required"}
+                ></TextField>
+              )}
+            />
+            <Button
+              type="submit"
+              sx={{
+                mt: 2,
+                background: theme.palette.primary.main,
+                color: "white",
```

Comment:
the default should already be white, can you send a screenshot of this button?

Reply:
if i do `color:theme` its a white button with green text, but in my head a green button with white text looks better

Reply:
by default with `variant='contained'` the button should be green with white text

Reply:
we can sort this later, its white button now idk what to tell you. ive started on the react-quill and idk if this stuff will stay but ill need to strip and recycle some of the code

Reply:
lol thats fair, i'll make a change in develop